pkg/kgo/config.go

```diff
@@ -1536,6 +1544,22 @@ func ConsumeRegex() ConsumerOpt {
 	return consumerOpt{func(cfg *cfg) { cfg.regex = true }}
```

Comment:
Can you add a note on ConsumeRegex:

```
// ConsumeRegex sets the client to parse all topics passed to [ConsumeTopics] as
// regular expressions. You can further use [ConsumeExcludeTopics] to exclude
// topics that would match any ConsumeTopics regex.
```

(If you can think of better wording, feel free)